js/main.js

```diff
@@ -43,3 +51,124 @@ hamBttn.addEventListener('click', () => {
     });
   }
 });
+
+// Projects popup
+const popupDivBG = document.createElement('div');
+const popupDiv = document.createElement('div');
+const projectBttns = document.querySelectorAll('.popup');
+const projectDiv = document.createElement('div');
+const myProjects = [{
+  name: 'Tonic',
+  imgUrl: './Images/pages/SnapshootPortfolio.png',
+  techs: ['html', 'css', 'ruby'],
+  description: 'A daily selection of privately personalized reads; no accounts or'
+  + 'sign-ups required.',
+  button: 'See Project',
+},
+{
+  name: 'Multi-post Stories',
+  imgUrl: './Images/pages/SnapshootPortfolio1.png',
+  techs: ['html', 'css', 'ruby'],
+  description: 'A daily selection of privately personalized reads; no accounts or'
+  + 'sign-ups required.',
+  button: 'See Project',
+},
+{
+  name: 'Tonic',
+  imgUrl: './Images/pages/SnapshootPortfolio2.png',
+  techs: ['html', 'css', 'ruby'],
+  description: 'A daily selection of privately personalized reads; no accounts or'
+  + 'sign-ups required.',
+  button: 'See Project',
+},
+{
+  name: 'Multi-post Stories',
+  imgUrl: './Images/pages/SnapshootPortfolio3.png',
+  techs: ['html', 'css', 'ruby'],
+  description: 'A daily selection of privately personalized reads; no accounts or'
+  + 'sign-ups required.',
+  button: 'See Project',
+},
+];
+
+function createWorksSection() {
+  if (myProjects.length !== 0) {
+    const myWorksSection = document.querySelector('.myWorksSection');
+    for (let i = 0; i < myProjects.length; i += 1) {
+      let technologies = '';
```

Comment:
- [ ] Kindly use `arrow functions` over the `function` keyword. Refer to the previous requirement i have given a detailed explanation of this